stream-engine/src/main/java/controller/affinity/SequentialBinding.java

```diff
@@ -21,7 +21,7 @@ public static int next_cpu() {
         ArrayList<Integer> list = mapping_node[socket];
         Integer core = list.get(cpu);
         cpu++;
-        if (cpu == 24) {//assume one socket 24 cores.
+        if (cpu == 19) {//assume one socket 24 cores.
```

Comment:
There should be 24 cores

Reply:
what does this mean? So you need to change 19 back to 24?